### PR TITLE
Added Seeder for testing

### PR DESCRIPTION
**Pull request by sujata-m**
**What this PR does ?**

- Create an organization with random org data
- Create a flex service
- Create a pathways station
- Create  4 random users with below - 
    - Role **poc**
    - Role **flex_data_generator**
    - Role **pathways_data_generator**
    - Role **osw_data_generator**
- Activate Organisation
- De-activate Organisation

**How to use this ?**

```
import {Seeder} from './seeder'

const run = async () => {
    const seeder = new Seeder()
    const seedData = await seeder.seed() 
    // this will return the seed data
    /*
    {
       "tdei_org_id":"4fd7c367-d7c3-49c3-9c7e-8e8ec6056169",
       "service_id":"fc7afae7-b406-4b36-af83-621aab026d8c",
       "station_id":"4aeb1717-c651-4d27-8ee5-34fdcf199bfa",
       "users":{
          "poc":{
             "username":"susan3@gmail.com",
             "password":"Pa$s1word"
          },
          "flex_data_generator":{
             "username":"taya.kertzmann@gmail.com",
             "password":"Pa$s1word"
          },
          "pathways_data_generator":{
             "username":"emely.boehm93@hotmail.com",
             "password":"Pa$s1word"
          },
          "osw_data_generator":{
             "username":"cornelius45@yahoo.com",
             "password":"Pa$s1word"
          }
   }
   */
   // To De-activate the Organisation
   const isDeactivated = await seeder.deactivateOrg('4fd7c367-d7c3-49c3-9c7e-8e8ec6056169')

   // To Activate the Organisation
   const isActivated = await seeder.activateOrg('4fd7c367-d7c3-49c3-9c7e-8e8ec6056169')
   
}

run()
```

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -140,35 +143,168 @@ export class Utility {
...
+    static getServiceUpload(orgId: string) {
+        return {
+            tdei_org_id: orgId,
+            service_name: `${faker.animal.cat()} Service`,
+            polygon: this.getRandomPolygon()
+        }
+    }
+}
+
+export class APIUtility {
```
Comment (susrisha) [inline src/utils.ts:215]: Move all these methods to seeder class itself.

File: src/seeder.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    private readonly roles: Array<string>
+    public seedData: {} | SeedData
+
+    constructor() {
+        this.client = new APIUtility()
+        this.roles = ['poc', 'flex_data_generator', 'pathways_data_generator', 'osw_data_generator']
+        this.seedData = {}
+    }
+
+    public async seed() {
```
Comment (susrisha) [inline src/seeder.ts:15]: this should return a Promise of type SeedData

**Review by susrisha — CHANGES_REQUESTED** [review]

File: src/utils.ts [hunk 3]
```diff
@@ -140,35 +143,168 @@ export class Utility {
...
+                method: 'post',
+                url: '/api/v1/station',
+                data: Utility.getStationUpload(orgId)
+            })
+            return resp?.data?.data
+        } catch (err: any) {
+            throw err?.response?.data?.message
+        }
+    }
+
```
Comment (susrisha) [inline src/utils.ts:296]: remove Org or deactivate Org call is missing

Comment (sujata-m) [reply src/utils.ts]: oohh! that i missed

**Pull request merged**